crates/solidity/outputs/cargo/crate/src/main.rs

```diff
@@ -5,20 +5,22 @@ use clap::{Parser as ClapParser, Subcommand};
 use semver::Version;
 use slang_solidity::{syntax::parser::ProductionKind, Language};
 
-mod _supress_library_dependencies_ {
-    // Below are dependencies used by the library `lib.rs`, but not here.
-    // However, we need to add a fake usage below to supress Cargo warnings about unused dependencies.
-    // This is a known issue, and we should remove this hack once there is a better solution from Cargo.
-    // https://github.com/rust-lang/cargo/issues/1982
+// Below are dependencies used by the library `lib.rs`, but not here.
```

Comment:
I suggest undoing changes to this file, and keeping the comments inside the suppression module, as it improves readability when using folding.